src/models/treemodel.cpp

```diff
@@ -74,6 +77,9 @@ QVariant BottomUpModel::rowData(const Data::BottomUp* row, int column, int role)
         if (role == SortRole) {
             return m_results.costs.cost(column - NUM_BASE_COLUMNS, row->id);
         }
+        if (m_diffMode && column - NUM_BASE_COLUMNS % 2 == 1) {
```

Comment:
add parens to clarify what the modulus operates on - it should be `(column - NUM_BASE_COLUMNS) % 2 == 1` or?